Git.md

```diff
@@ -1,36 +1,47 @@
-
-## FreeCodeCamp's Wiki Resources
-
-* [Ammend Last Commit](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Amend-Last)
-* [Blame](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Blame)
-* [Change URL of a remote repository](https://github.com/FreeCodeCamp/wiki/blob/master/Git-Change-the-URL-of-a-remote-repository.md)
-* [Checkout Remote Branch](https://github.com/FreeCodeCamp/wiki/blob/master/Git-Checkout-A-Remote-Branch.md)
-* [Delete a Git branch both locally and remotely](https://github.com/FreeCodeCamp/wiki/blob/master/Git-Delete-a-Branch-both-locally-and-remotely.md)
-* [Force Git to overwrite local files on Pull](https://github.com/FreeCodeCamp/wiki/blob/master/Git-Force-Git-to-overwrite-local-files-on-pull.md)
-* [Merge](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-merge)
-* [Pull Vs Fetch](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Pull-Vs-Git-Fetch)
-* [Push Local to Remote Repository](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Push-Local-To-Remote-Repository)
-* [Rebase](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase) 
-* [Shortcut](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Shortcut)
-* [Undo a Commit and Redo](https://github.com/FreeCodeCamp/wiki/blob/master/Git-Undo-Redo-A-Commit.md)
-
-## Books and Tutorials
-- [Pro Git](https://git-scm.com/book) - The entire Pro Git book, written by Scott Chacon and Ben Straub and published by Apress, is available here. Download this book in PDF, mobi, or ePub form for free.
-- [Become a git guru - Atlassian](https://www.atlassian.com/git/tutorials/) - Responsive site with Git tutorials, training and articles.
-- [Git In The Trenches](http://cbx33.github.io/gitt/) - Git In The Trenches, or GITT is designed to be a book that focusses on teaching people to use Git by associating with scenarios that are experienced by a fictional company called Tamagoyaki Inc. Download this book in PDF, mobi, or ePub form for free.
-- [Official Git Tutorial](https://git-scm.com/docs/gittutorial) - This tutorial explains how to import a new project into Git, make changes to it, and share changes with other developers.
-- [Official Git User Manual](https://git-scm.com/docs/user-manual.html) - This manual is designed to be readable by someone with basic UNIX command-line skills, but no previous knowledge of Git.
-- [Try Git Tutorial by Github and CodeSchool](https://try.github.io) - This tutorial is a quick 15 minutes sprint to get started with Git within the browser.
-
-
-## Other Resources
-- [Git Ready](http://gitready.com) - 'Learn git one commit at a time' by Nick Quaranto
-- [Hub](https://hub.github.com/) - Hub is a command-line wrapper for git that makes you better at GitHub.
+# Git Resources
+
+A collection of resources on the [Git VCS](https://git-scm.com).
+
+```
+$ git --distributed-is-the-new-centralized
+```
```

Comment:
Please remove this completely.
```
    ```
    $ git --distributed-is-the-new-centralized
    ```
```